preflight-checks/src/checks/03-docker-build.js

```diff
@@ -0,0 +1,102 @@
+import chalk from 'chalk';
+import packageJson from '../package';
+import execa from "execa";
+import findUp from 'find-up';
+
+import fs from 'fs';
+import process from 'process';
+
+const chalkVIPGo = chalk.yellow( '@automattic/vip-go' );
+const chalkPackageJson = chalk.yellow( 'package.json' );
+
+const envVariables = {
+	'VIP_GO_APP_ID': '123',
```

Comment:
Let's also add a few more env vars that we supply and use a clearer "fake" site ID:

```suggestion
	'VIP_GO_APP_ID': 'unknown',
	'VIP_GO_APP_NAME': 'unknown',
	'VIP_GO_APP_ENVIRONMENT': 'local',
```